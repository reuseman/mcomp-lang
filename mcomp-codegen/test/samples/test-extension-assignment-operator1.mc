component EntryPoint provides App { 
  def main() : int {
    var i : int;

    i = 0;
    print(i);
    
    i += 3;
    print(i);

    i *= 3;
    print(i);

    i -= 2;
    print(i);

    i /= 2;
    print(i);

    i %= 2;
    print(i);

    return 0;
  }
}