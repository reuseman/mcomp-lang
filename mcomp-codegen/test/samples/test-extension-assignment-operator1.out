0
3
9
7
3
1
